client/my-sites/stats/style.scss

```diff
@@ -18,6 +18,14 @@
 	margin-left: 4px;
 }
 
+.is-section-stats .followers-count .button {
```

Comment:
I then realized that the file has no `stats-navigation` class to use. The whole stats tree lacks a wrapper class — we should have them in the different `Mains` at the very least — so the most appropriate at the moment is the main stats stylesheet.